tests/foreman/cli/test_contentview.py

```diff
@@ -3962,3 +3962,23 @@ def test_positive_inc_update_should_not_fail(self, module_org):
         assert result[2]
         content_view = ContentView.info({'id': content_view['id']})
         assert '1.1' in [cvv_['version'] for cvv_ in content_view['versions']]
+
+    @pytest.mark.tier1
+    def test_positive_pulp_workers_running_state(self, target_sat):
+        """Ensure that 6 pulpcore-workers are in active (running) state
+
+        :BZ: 1848111
+
+        :id: f0b8b354-1a30-41c3-93d7-fb51e99c82b6
+
+        :customerscenario: true
+
+        :expectedresults: 6 pulpcore-workers are in active (running) state
+
+        :SubComponent: Pulp
+        """
+        output = target_sat.execute('systemctl status pulpcore-worker* | grep Active')
+        result = output.stdout.split('\n')
+        assert len(result) - 1 == 6
```

Comment:
Why assert against 6 specifically? Is there an installer default tied to this assumption?  Any way to check config/answer/log files for what the expected number of pulpcore-worker services is? Cursory search didn't find anything, other than the service definition being dynamic.

There are really a few assertions going on here in addition to the BZ, its good. Combined with the other service test for pulp, I think you could improve them as well by checking their run state in addition to configuration.

Suggestion for a bit more pythonic way to make this assertion (along with the above change to using rstrip).

```
# this might not fit on 100 chars, break up if necessary. rstrip + splitlines instead of explicitly using `\n`.

output = sat.execute('systemctl status pulpcore-worker* |grep "Active:"').stdout.rstrip().splitlines()
assert all('Active: active (running)' in r for r in output)
```

Reply:
pulpcore-worker count is set by the installer flag and currently, it is set to 6 because of the number of CPU cores used. We probably shouldn't keep this number constant in this test, so rstrip should be useful here.